meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -104,6 +111,19 @@ function runhacks {
         log "Save timestamp..."
         date -u +%4Y%2m%2d%2H%2M%2S > /etc/timestamp
     fi
+
+    # Some old devices didn't have curl which we need for supervisor update
+    # If that's the case, replace by wget usage
+    if which curl &>/dev/null; then
+        log "Curl hack: Curl in place."
+    else
+        script_path="/usr/bin/update-resin-supervisor"
+        if ! [ -e $script_path ]; then
```

Comment:
It's just a sanity check if we move the script in other paths or not use it the future.